lookbook/previews/op_primer/email_updates_mode_selector_component_preview.rb

```diff
@@ -0,0 +1,117 @@
+# frozen_string_literal: true
+
+#-- copyright
+# OpenProject is an open source project management software.
+# Copyright (C) the OpenProject GmbH
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License version 3.
+#
+# OpenProject is a fork of ChiliProject, which is a fork of Redmine. The copyright follows:
+# Copyright (C) 2006-2013 Jean-Philippe Lang
+# Copyright (C) 2010-2013 the ChiliProject Team
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+# See COPYRIGHT and LICENSE files for more details.
+#++
+
+module OpPrimer
+  # @logical_path OpenProject/Primer
+  class EmailUpdatesModeSelectorComponentPreview < Lookbook::Preview
+    # See the [component documentation](/lookbook/pages/components/email_updates_mode_selector) for more details.
+    # @param toggle [Boolean]
+    # @param path [String]
+    # @param title [String]
+    # @param enabled_description [String]
+    # @param disabled_description [String]
+    # @param alt_text [String]
+    # @param show_button [Boolean]
+    # @param method [Symbol] select [get, post, patch]
+    def playground(
+      toggle: true,
+      path: "/projects/demo-project/meetings/113/toggle_notifications",
+      title: "Sidebar component title",
+      enabled_description: "Email updates are enabled for all users.",
+      disabled_description: "Email updates are disabled for all users.",
+      alt_text: "To change this, edit the series template.",
+      show_button: true,
+      method: :patch
+    )
+      render OpPrimer::EmailUpdatesModeSelectorComponent.new(
+        toggle:,
+        path:,
+        title:,
+        enabled_description:,
+        disabled_description:,
+        alt_text:,
+        show_button:,
+        method:
+      )
+    end
+
+    def enabled(toggle: true, path: "", title: "Sidebar component title",
+                enabled_description: "Email updates are enabled for all users.",
+                disabled_description: "Email updates are disabled for all users.")
+      render OpPrimer::EmailUpdatesModeSelectorComponent.new(
+        toggle:,
```

Comment:
I know that this is the ruby way. However, this is what is shown in the notebook notes so in the end nobody knows what she passed here to achieve that the component is enabled. I suggest that you do not pass these things as parameters because in these examples they do not have to be configurable instead pass the attribute directly to the component.
<img width="538" height="232" alt="Bildschirmfoto 2025-08-06 um 14 00 55" src="https://github.com/user-attachments/assets/ecd528e6-13de-4d5a-8489-a7d580a4914f" />